src/components/Media.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { apiKey, domainName } from '../utils/apiUrls';
+import { posterUrlBig } from '../utils/apiUrls';
+import MediaPage from './media_page/MediaPage';
+import averageColor from '../utils/averageColor';
+import Loading from './Loading';
+const setStyle = document.documentElement.style;
+
+export default function Movie() {
+    // Requiring information about the media
+    const location = useLocation();
+    const data = location.state;
+    const { mediaType, id} = data;
+
+    // Constructing the media link to fetch its information from the Api
+    const path = `${domainName}${mediaType}/${id}?api_key=${apiKey}&language=en-US`;
+    const [info, setInfo] = useState(0);
```

Comment:
If I am not mistaken the returned object is an array. The initial should be an empty array.
```suggestion
    const [info, setInfo] = useState([]);
```